server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -760,6 +778,8 @@ protected QueryDataSet processShowQuery(ShowPlan showPlan, QueryContext context)
         return processShowPathsSetSchemaTemplate((ShowPathsSetTemplatePlan) showPlan);
       case PATHS_USING_SCHEMA_TEMPLATE:
         return processShowPathsUsingSchemaTemplate((ShowPathsUsingTemplatePlan) showPlan);
+      case MIGRATION:
```

Comment:
Please rename MIGRATION to SHOW_MIGRATION.

Reply:
Fixed